File: 1000/118A_Sring_Task.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string cad; cin >> cad;
    string resul = ""; 
    for (char c : cad) {
        c = tolower(c);
        if (c != 'a' && c != 'o' && c != 'y' && c != 'e' && c != 'u' && c != 'i') {
            resul += '.';
            resul += c;
        }
    }
    cout << resul;
    return 0;
}

File: 1000/118A_Sring_Task.py
cad = list(input())
resul = ""

for c in cad:
    c = c.lower()
    if c != 'a' and c != 'o' and c != 'y' and c != 'e' and c != 'u' and c != 'i':
        resul += '.'
        resul += c

print(resul)
